tests/testthat/test-basic.r

```diff
@@ -158,7 +158,12 @@ test_that("toStringNET works for primitive types", {
 })
 
 # Re-enable when https://github.com/Open-Systems-Pharmacology/rSharp/issues/35 is fixed
```

Comment:
Take that comment out now.

Reply:
ok, also, when running this test I'm not sure how it's counted. I don't see an assert within ```test_that```, so I'm suspicious that it is ignored or skipped or whatever. This is how the test came to you, but see if you can find out how to make this test count as a pass/fail.

Also, don't print anything on the screen if the test passes. Right now I'm seeing some stuff in my RStudio. It's mixed in with all the obsolete stuff too, which @PavelBal is removing in one of his upcoming PR's

```
 Type:    System.Exception
Message: An exception designed with a particular stack trace length
Method:  Void ThrowException(Int32)
Stack trace:
   at ClrFacade.TestCases.ThrowException(Int32 stackDepth)
   at ClrFacade.TestCases.ThrowException(Int32 stackDepth)
   at ClrFacade.TestCases.ThrowException(Int32 stackDepth)
   at ClrFacade.TestCases.ThrowException(Int32 stackDepth)
   at ClrFacade.TestCases.ThrowException(Int32 stackDepth)
   at ClrFacade.TestCases.ThrowException(Int32 stackDepth)
   at ClrFacade.TestCases.ThrowException(Int32 stackDepth)
   at ClrFacade.TestCases.ThrowException(Int32 stackDepth)
   at ClrFacade.TestCases.ThrowException(Int32 stackDepth)
   at ClrFacade.TestCases.ThrowException(Int32 stackDepth)
   at System.RuntimeMethodHandle.InvokeMethod(Object target, Void** arguments, Signature sig, Boolean isConstructor)
   at System.Reflection.MethodBaseInvoker.InvokeDirectByRefWithFewArgs(Object obj, Span`1 copyOfArgs, BindingFlags invokeAttr)
```

Reply:
Don't care about R stuff at this stage.